src/java/components/gpiomanager/src/test/java/hu/bme/mit/inf/modes3/components/gpiomanager/TestGpioWriter.java

```diff
@@ -0,0 +1,35 @@
+package hu.bme.mit.inf.modes3.components.gpiomanager;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class TestGpioWriter implements CommandWriter {
```

Comment:
What is the reason for having a 'dedicated' `TestGpioWriter` class instead of just mocking the corresponding field by mockito in the unit tests?

Reply:
True, it can be refactored.